device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -1142,6 +1151,10 @@ public Object call()
 
                 this.listener.onConnectionLost(transportException, connectionId);
             }
+            finally
+            {
+                iotHubReactor.free();
```

Comment:
Why no `this.` in this case?

Reply:
No reason, I suppose. We don't really have strict guidelines on if we use them or not in Java